tqdm/_tqdm.py

```diff
@@ -402,8 +402,14 @@ def inner(df, func, *args, **kwargs):
                 # Precompute total iterations
                 total = getattr(df, 'ngroups', None)
                 if total is None:  # not grouped
-                    total = len(df) if isinstance(df, Series) \
-                        else df.size // len(df)
+                    if isinstance(df, Series):
+                        total = len(df)
+                    elif kwargs.get('axis', 0) == 0:
```

Comment:
Simply:
```
elif kwargs.get('axis', 0) == 0 or args[0] == 0:
```

Because the other arguments are not integers. But yeah it's not foolproof, but I cannot see any better way to handle that.